client/src/App.js

```diff
@@ -1,18 +1,18 @@
 import React from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import ExamplePage from "./pages/ExamplePage";
+import Map from "./components/Map";
 
 function App() {
   return (
     <div className="App">
```

Comment:
make sure to add some sort of aria attributes -- i think maybe role="main" might work?